### DISCUSSION TITLE
nvim/treesitter installation

### DISCUSSION THREAD

**Discussion by jamaza7** [LazyVim/LazyVim#530] [Q&A]
Hello everyone, i am currently experimenting with nvim, i am a newbie and when i installed lazy i found this error when opening nvim.

- ### Installation not possible: ...vim/lazy/nvim-treesitter/lua/nvim-treesitter/install.lua:86: Parser not available for language "help".

**Comment by dpetka2001** [comment]
Plz see [this issue](https://github.com/LazyVim/LazyVim/issues/524) where it has been discussed.

**Reply by jamaza7** [reply]
yes I was able to solve it thank you
